ad_clicks/projects/ad_clicks_project_1.py

```diff
@@ -0,0 +1,157 @@
+
+# https://7b7qhlx1yh.execute-api.us-east-1.amazonaws.com/Predict/cf09696a-e6f6-482a-ac75-fd1ad61772a4
+
+import requests
+import json as json
+import numpy as np
+
+import argparse
+import tkinter as tk
+import PySimpleGUI as sg
+
+
+# Sample format of data expected
+# {"Pregnancies":6,"Glucose":148,"BloodPressure":72,"SkinThickness":35,
+# "Insulin":0,"BMI":33.6,"DiabetesPedigreeFunction":0.627,"Age":50}
+
+def mystr(inp_val, inp_feature):
+    tmp_val = inp_val
```

Comment:
check for type to be str and add quotes around it. 

Also name the function to be wrap_string_in_quotes